playwright/pages/pageElements/rightSideDetailsSection.ts

```diff
@@ -98,6 +100,18 @@ export class RightSideDetailsPage {
     await this.selectWarehouseShippingMethodButton.click();
   }
 
+  async expectAddedWarehousesVisibleByNames(names: string[]) {
+    for (const name of names) {
+      await this.warehouseSection.getByTestId(`#selected-option-${name}`);
```

Comment:
There is no expect in here, by design?

Reply:
[fixed](https://github.com/saleor/saleor-dashboard/blob/b4345344d7529077f8b9966d3e965994ab3c7ec4/playwright/pages/pageElements/rightSideDetailsSection.ts#L104)